diagnostic.md

```diff
@@ -21,13 +21,18 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --request GET http://www.example.com/donuts
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function() {
+    $.ajax({
```

Comment:
don't forget to prevent your default page action but passing in event and using: event.preventDefault();